examples/cgo/cc_dependency/BUILD

```diff
@@ -0,0 +1,36 @@
+config_setting(
+    name = "darwin",
+    values = {"cpu": "darwin"},
+)
+
+cc_library(
+    name = "version",
+    srcs = ["cxx_version.cc"],
+    hdrs = ["version.h"],
+    # TODO(yugui) Support darwin too and remove this workaround.
+    # See also comments in cxx_version.cc.
```

Comment:
No, as I described in `cxx_version.cc`, `osx_cc_wrapper.sh` is the source of the problem here.

Without this `select`, bazel would add several instances of `-Wl,-rpath` into the command line. But `osx_cc_wrapper.sh` expects only one `-Wl,-rpath`.  It is the problem.